containers/podman-desktop
# chore: enable eslint in the svelte files

### What does this PR do?
There is a new svelte plugin for eslint parsing AST and providing a parser, so we can use this new plugin.
But as there are a lot of reports, for now, turn off almost all the "projects rules" as there are like 500+ errors will uncomment/fix rule by rule

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

fixes #2812 

### How to test this PR?

PR check should be green (no-op for now as code is still the same)

Change-Id: I42fc9d68633f7e534e726032eddaae858e5d832c
